packages/api-explorer/src/components/SelectorContainer/SelectorContainer.tsx

```diff
@@ -51,6 +51,9 @@ export const SelectorContainer: FC<SelectorContainerProps> = ({
   ...spaceProps
 }) => {
   const searchParams = new URLSearchParams(location.search)
+  // TODO: noticing that there are certain pages where we must delete extra params
+  //       before pushing its link, what's a way we can handle this?
```

Comment:
If this was not included here, navigating to the diffScene after selecting a filter will persist that parameter in the URL. I worry that there will be a growing number of scenes to render where certain  parameters are no longer necessary and must be explicitly removed/nullified to avoid bloating the URL. 
- What if we include an optional `toRemove` parameter to the `buildPath` functions, which is a list of strings corresponding to search params which we will remove from the URL? This could clean up changes in `SideNavMethods` and `SideNavTypes`.